src/ui/templates/insurance/policy/bank-details.njk

```diff
@@ -0,0 +1,151 @@
+      <span class="govuk-caption-xl" data-cy="heading-caption">{{ CONTENT_STRINGS.HEADING_CAPTION }}</span>
+
+      <h1 class="govuk-heading-xl" data-cy="{{ DATA_CY.HEADING }}">{{ CONTENT_STRINGS.PAGE_TITLE }}</h1>
+
+      {{ govukHint({
+        text: CONTENT_STRINGS.HINT,
+        classes: "govuk-!-margin-bottom-5",
+        attributes: {
+          "data-cy": "bank-details-hint"
+        }
+      }) }}
+    </div>
+  </div>
+
+  <form method="POST" data-cy="form" enctype="application/x-www-form-urlencoded" novalidate>
+
+    <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-three-quarters-from-desktop">
+
+        {{ govukInput({
+          label: {
+            text: FIELDS.SORT_CODE.LABEL,
+            classes: "govuk-!-font-weight-bold govuk-!-font-size-24",
+            attributes: {
+              id: FIELDS.SORT_CODE.ID + '-label',
+              'data-cy': FIELDS.SORT_CODE.ID + '-label'
+            }
+          },
+          hint: {
+            text: FIELDS.SORT_CODE.HINT,
+            attributes: {
+              'data-cy': FIELDS.SORT_CODE.ID + '-hint'
+            }
+          },
+          classes: "govuk-input--width-10",
+          attributes: {
+            'data-cy': FIELDS.SORT_CODE.ID + '-input'
+          },
+          id: FIELDS.SORT_CODE.ID,
+          name: FIELDS.SORT_CODE.ID,
+          value: submittedValues[FIELDS.SORT_CODE.ID] or application.policy[FIELDS.SORT_CODE.ID],
+          errorMessage: validationErrors.errorList[FIELDS.SORT_CODE.ID] and {
+            text: validationErrors.errorList[FIELDS.SORT_CODE.ID].text,
+            attributes: {
+              "data-cy": FIELDS.SORT_CODE.ID + "-error-message"
+            }
+          }
+        }) }}
+
+        {{ govukInput({
+          label: {
+            text: FIELDS.ACCOUNT_NUMBER.LABEL,
+            classes: "govuk-!-font-weight-bold govuk-!-font-size-24",
+            attributes: {
+              id: FIELDS.ACCOUNT_NUMBER.ID + '-label',
+              'data-cy': FIELDS.ACCOUNT_NUMBER.ID + '-label'
+            }
+          },
+          hint: {
+            text: FIELDS.ACCOUNT_NUMBER.HINT,
+            attributes: {
+              'data-cy': FIELDS.ACCOUNT_NUMBER.ID + '-hint'
+            }
+          },
+          classes: "govuk-input--width-10",
+          attributes: {
+            'data-cy': FIELDS.ACCOUNT_NUMBER.ID + '-input'
+          },
+          id: FIELDS.ACCOUNT_NUMBER.ID,
+          name: FIELDS.ACCOUNT_NUMBER.ID,
+          value: submittedValues[FIELDS.ACCOUNT_NUMBER.ID] or application.policy[FIELDS.ACCOUNT_NUMBER.ID],
+          errorMessage: validationErrors.errorList[FIELDS.ACCOUNT_NUMBER.ID] and {
+            text: validationErrors.errorList[FIELDS.ACCOUNT_NUMBER.ID].text,
+            attributes: {
+              "data-cy": FIELDS.ACCOUNT_NUMBER.ID + "-error-message"
+            }
+          }
+        }) }}
```

Comment:
Have been wanting to create a new `textInput` type nunjucks component because all instances are almost exactly the same except for the field ID and e.g custom classes.

Not a blocker, I think we should create a ticket that blocks the page setup for international bank details. Until we have this component, the more pages that are built, the longer the task will take